unison-cli/src/Unison/CommandLine/InputPatterns.hs

```diff
@@ -1020,7 +1043,7 @@ back =
     []
     ( P.wrapColumn2
         [ ( makeExample back [],
-            "undoes the last" <> makeExample' cd <> "command."
+            "undoes the last" <> makeExample' projectSwitch <> "or" <> makeExample' cd <> "command."
```

Comment:
Up to you, but if we're deprecating CD anyways, might not make sense to mention it in the docs. I can see the argument either way though :) 

```suggestion
            "undoes the last" <> makeExample' projectSwitch <> "command."
```